### DISCUSSION TITLE
AWS CodeBuild with serverless-artillery

### DISCUSSION THREAD

**Discussion by arthurflame** [artilleryio/artillery#1140] [Artillery]
I'm using AWS CodeBuild to run serverless-artillery, and I'm using a csv file as a payload.
This csv file is generated with a python script and is saved in the current working directory, so the path in artillery script is specified like this  `payload: - path: "names.csv"`, but artillery appends a random directory(?) where it is looking for this file, I have tried to create this directory and put the file there, but still it couldn't find the csv file.
### Serverless-artillery log:
![image](https://user-images.githubusercontent.com/43091238/128005194-f35a0893-8f2a-4d9c-8f0e-60a86377fa24.png)
Any suggestions where/how this file should be stored when using custom payloads for artillery using CodeBuild?

**Comment by hassy** [comment]
hi @arthurflame, serverless-artillery is a third-party project built on top of Artillery, we don't provide support for it.

Looking at the stack trace, I would guess that serverless-artillery does not support using external CSV files. `/var/task` is the directory in which a lambda executes, and the CSV file is missing, i.e. serverless-artillery did not bundle it properly.

**Reply by arthurflame** [reply]
Thank you for your response, found out what was the issue about.
I was generating the `names.csv` file during the invocation of the serverless-artillery, during this phase I downloaded the .zip file that was generated by initial serverless-artillery deployment and, since the `names.csv` was missing from there and was generated only during the invocation, it couldn't find it, but after I've placed it into that zip, it was successfully found.
Thank you.
